ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayAnnotation/DisplayAnnotation.qml

```diff
@@ -0,0 +1,60 @@
+// [WriteFile Name=DisplayAnnotation, Category=Layers]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapLightGrayCanvas {}
+
+            FeatureLayer {
+                ServiceFeatureTable {
+                    id: featureTable
+                    url: "https://services1.arcgis.com/6677msI40mnLuuLr/arcgis/rest/services/East_Lothian_Rivers/FeatureServer/0"
+                }
+            }
+
+            AnnotationLayer {
+                ServiceFeatureTable {
+                    id: annotationFeatureTable
+                    url: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/RiversAnnotation/FeatureServer/0"
+                }
+            }
+
+            initialViewpoint:
+                ViewpointCenter {
```

Comment:
If you look at the ArcGIS Qt QML docs, a type of `ViewpointCenter` is automatically bound to `initialViewpoint` if declared within `AnnotationLayer`. Remove the `initialViewpoint:` line and fix the tabbing.

Reply:
This is using a concept in QML called default properties - https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#default-properties